setup.py

```diff
@@ -13,6 +13,7 @@
     'pycurl==7.19.0',
     'gunicorn==0.17.4',
     'ipaddress==1.0.4',
+    'netaddr==0.7.11',
```

Comment:
Change this to netaddr>=0.7.11 and I think I can accept the pull request.  Going forward, I only want minimum requirements.